### PR TITLE
update lightning to >=1.9, bump other minimums, stage 0.20.0b1

### PR DESCRIPTION
**Pull request by adamgayoso**

### CODE REVIEW

File: scvi/__init__.py [hunk 1]
```diff
@@ -3,12 +3,6 @@
 # Set default logging handler to avoid logging with logging.lastResort logger.
 import logging
 
-try:
-    # necessary as importing scvi after ray causes kernel crash
-    from ray import tune  # noqa
-except ImportError:
-    pass
-
```
Comment (martinkim0) [inline scvi/__init__.py:11]: Does it work now without this block?

Comment (adamgayoso) [reply scvi/__init__.py:11]: I was having no issues. Can you check as well?

Comment (martinkim0) [reply scvi/__init__.py:11]: Yeah I'm still having the issue unfortunately. I usually check by running
```
import scvi
import ray
ray.init()
ray.shutdown()
```

Comment (adamgayoso) [reply scvi/__init__.py:11]: I'm not having that issue and the tests are passing, I'm wondering what could be happening

**Review by martinkim0 — APPROVED** [review]

**Pull request merged**